upstash_vector/core/index_operations.py

```diff
@@ -34,9 +45,15 @@ def _execute_request(self, payload, path):
     def upsert(
         self,
         vectors: Sequence[Union[Dict, tuple, Vector, Data]],
+        namespace: str = DEFAULT_NAMESPACE,
```

Comment:
could also pass this in __init__ maybe but not really important

it could be like
```
# usage 1
index = Index()
index.upsert() # namespace=""
index.upsert(namespace="foo") # namespace="foo"

# usage 2
index = Index(namespace="bar")
index.upsert() # namespace="bar"
index(namespace="baz") # namespace="baz"
```

Reply:
I would prefer to add a separate method like ts client if we want to do that in the future (index.namespace("foo"))